.github/workflows/release_pytorch_wheels.yml

```diff
@@ -7,14 +7,25 @@ permissions:
   contents: read
 
 jobs:
-  build_pytorch_grid:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      version_suffix: ${{ steps.set.outputs.version_suffix }}
+    steps:
+      - id: set
+        run: echo "version_suffix=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
```

Comment:
TODO: move to script or reusable workflow? We should centralize all of our version handling.